sys-utils/lscpu-arm.c

```diff
@@ -116,6 +118,14 @@ static const struct id_part cavium_part[] = {
     { 0x0a2, "ThunderX-81XX" },
     { 0x0a3, "ThunderX-83XX" },
     { 0x0af, "ThunderX2-99xx" },
+    { 0x0b0, "OcteonTX2" },
+    { 0x0b1, "OcteonTX2-98XX" },
+    { 0x0b2, "OcteonTX2-96XX" },
+    { 0x0b3, "OcteonTX2-95XX" },
+    { 0x0b4, "OcteonTX2-95XXN" },
+    { 0x0b5, "OcteonTX2-95XXMM" },
+    { 0x0b6, "OcteonTX2-95XXO" },
+    { 0x0b8, "ThunderX3T110" },
```

Comment:
Would it be better to add a hyphen to the string, it means "ThunderX3-T110"?

Reply:
Thought about that as well and searched various combinations combined with `site:marvell.com`. There neither "ThunderX3T110", "ThunderX3-T110" and not even "T110" can be found and vast majority of other web search hits shows "ThunderX3T110" only.

Now thinking of it that's mostly commits to build tools and they seem to avoid hyphens in those names for whatever reasons. So for consistency it might be better to switch to "ThunderX3-T110". I'll add another minor commit...